pages/metrics.py

```diff
@@ -19,6 +19,9 @@
 userDataObject = auth.get_user_app_id_proto()
 lister = ClarifaiResourceLister(stub, auth.user_id, auth.app_id, page_size=page_size)
 
+st.session_state['total'] = 0
+st.session_state.get_input_count_response = {}
```

Comment:
Not sure what should be the initial values of these session variables

Reply:
that seems fine